blueprints/common/partials/transfer_user.hbs

```diff
@@ -0,0 +1,56 @@
+{{!--
+  Creates an Image Builder Image
+https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-image.html
+--}}
+
+"tu{{sanitize this.Name ''}}": {
+  "Type": "AWS::Transfer::User",
+  "Properties": {
+
+    "Role": "{{ this.Role }}",
+
+    "ServerId": {{#ifObject this.ServerId}}
+        { {{{inject this.ServerId}}} }
+      {{else}}
+        { "Ref":"ts{{this.ServerId}}" }
```

Comment:
You need to sanitize anything that references sanitized IDs.
`{ "Ref":"ts{{this.ServerId}}" }` --> `{ "Ref":"ts{{sanitize this.ServerId ''}}" }`